docs/source/contributing.rst

```diff
@@ -2,7 +2,7 @@
 Contributing
 ############
 
-Thank you for your interest in contributing to Manim! However you have decided to contribute
+Thank you for your interest in contributing to Manim! However, you have decided to contribute
```

Comment:
The original sentence was correct - in fact, this change makes the meaning incorrect.